packages/components/psammead-consent-banner/src/__snapshots__/index.test.jsx.snap

```diff
@@ -29,15 +34,13 @@ exports[`ConsentBanner should correctly render for ltr service 1`] = `
 
 .emotion-2 a {
   color: #F6A21D;
-  -webkit-text-decoration: underline;
-  text-decoration: underline;
-  -webkit-text-decoration-color: #AEAEB5;
-  text-decoration-color: #AEAEB5;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  border-bottom: solid 0.0625rem #AEAEB5;
 }
 
 .emotion-2 a:focus {
-  outline: none;
-  box-shadow: 0rem 0rem 0.0625rem 0.1875rem #68A1F8;
+  outline: solid #68A1F8;
```

Comment:
What is the colour for? Usually this is just the default styling from CSS e.g. we don't overide any defaults

Reply:
Also why solid?

Reply:
We shouldn't declare anything for outline...

Reply:
Ah I thought outline colour varied with browsers, so I implemented the colour used in the Zeplin, happy to change this. Is `outline: solid;` sufficient in this case then?

Reply:
We shouldn't declare anything for outline... and the browser will just do it's thing.